common/helpers/frameworks/set-validation-rules.test.js

```diff
@@ -0,0 +1,159 @@
+const helper = require('./set-validation-rules')
+
+describe('Framework helpers', function () {
+  describe('#setValidationRules', function () {
+    context('with no responses', function () {
+      it('should return original item', function () {
+        const mockField = ['mock-field', { foo: 'bar' }]
+        const output = helper()(mockField)
+
+        expect(output).to.deep.equal(['mock-field', { foo: 'bar' }])
+      })
+    })
+
+    context('with responses', function () {
+      let output
+      const mockResponses = [
+        {
+          id: '1',
+          question: {
+            key: 'violent-or-dangerous',
+          },
+        },
+      ]
+
+      context('without parent response', function () {
+        const mockField = ['mock-field', { foo: 'bar' }]
```

Comment:
Repeating the field made it easier to understand what was happening in each context :100: